src/Backend/Modules/Users/info.xml

```diff
@@ -10,6 +10,13 @@
     <module>Locale</module>
     <module>Settings</module>
   </depends_on>
+  <variables>
```

Comment:
what do you need this for?

Reply:
This is the list of variables that you want to initialize for the installation of the module.